gruntfile.js

```diff
@@ -244,6 +245,12 @@ gruntfile_cfg.replace = {
         flatten: true,
         src: ['.tmp/concat/js/app.js'],
         dest: '.tmp/concat/js'
+      },
+      {
+        expand: true,
```

Comment:
Why is this new section needed in gruntfile?

Reply:
This section used to prepare replace the "../webfonts" string in the css file for fontawesome v5, because FA v5 using "webfonts" path dir instead of "fonts".